context/app/static/js/components/home-revision/AssayTypeBarChart/AssayTypeBarChart.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useMemo } from 'react';
+import { scaleLinear, scaleOrdinal, scaleBand } from '@visx/scale';
+import { BarStackHorizontal } from '@visx/shape';
+import { Group } from '@visx/group';
+import { AxisTop, AxisLeft } from '@visx/axis';
+import { withParentSize } from '@visx/responsive';
+
+import { AppContext } from 'js/components/Providers';
+import useSearchData from 'js/hooks/useSearchData';
+import { formatAssayData, addSumProperty, sortBySumAscending } from './utils';
+import { useChartPalette } from './hooks';
+
+const organTypesQuery = {
+  size: 0,
+  aggs: {
+    organ_types: { terms: { field: 'origin_sample.mapped_organ.keyword', size: 10000 } },
+  },
+};
+
+const assayTypesQuery = {
```

Comment:
My preference would be to combine these and make fewer, larger ES queries... and down the road, the same query might get the facets we need for pseudo-search.

But not necessary right now, and I might be optimizing for the wrong thing... I guess I worry a bit about making lots of simultaneous requests, but if this makes the code simpler, might be worth it.